tools/doc/package.json

```
@@ -7,7 +7,7 @@
     "node": ">=14.8.0"
   },
   "devDependencies": {
-    "highlight.js": "^11.3.1",
+    "highlight.js": "^11.2.0",
```

You meant to pin the version, right?
```suggestion
    "highlight.js": "11.2.0",
```

---

I just partially undid the package updates in https://github.com/nodejs/node/pull/41036. I think the `package-lock.json` takes care of it as the `Makefile` runs `npm ci`:
https://github.com/nodejs/node/blob/a2572940cdff72cdaa361b78389f05e5b91de315/Makefile#L686-L691

---

This didn't start failing until https://github.com/nodejs/node/pull/41036 was merged -- the newer highlight.js was published two months ago: https://www.npmjs.com/package/highlight.js

---

FWIW https://ci.nodejs.org/job/node-test-commit-linux-containered/29828/nodes=ubuntu1804_sharedlibs_withoutintl_x64/ passed.